### ISSUE TITLE
installation on arch linux fails

### ISSUE THREAD

**Issue by AndreLeifert** [dbcli/litecli#197]
both packer and yay -S litecli-git result in
==> ERROR: A failure occurred in build().

It seems some python packages need to be installed prior to running yay (or packer).

**Comment by tomit4** [comment]
Yes, this has been broken for quite some time now on Arch Linux. Either The Arch repos have to update the setuptools as it seems to be on 1.75.2.0-4 as of this writing.

Or it's possible this package doesn't need a python-setuptools at this high of a version number when built with a python version at or above version 3.8. Here is the error message when attempted to install via paru:

```
ERROR Missing dependencies:
        setuptools-scm>=8; python_version >= "3.8"
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'litecli-1.13.2-2':
error: packages failed to build: litecli-1.13.2-2
```
